### PR TITLE
Process 2015 is coming

### PR DESCRIPTION
**Pull request by plehegar**
For #462 
Do not merge and deploy before September 1, 2015

### CODE REVIEW

File: js/w3c/headers.js [hunk 1]
```diff
@@ -405,8 +405,8 @@ define(
                 conf.dashDate = utils.concatDate(conf.publishDate, "-");
                 conf.publishISODate = utils.isoDate(conf.publishDate);
                 conf.shortISODate = conf.publishISODate.replace(/T.*/, "");
-                conf.processVersion = conf.processVersion || "2014";
-                conf.isNewProcess = conf.processVersion == "2014";
+                conf.processVersion = conf.processVersion || "2015";
+                conf.isNewProcess = conf.processVersion == "2015";
```
Comment (darobin) [inline js/w3c/headers.js]: This is wrong. Any document that has its `processVersion` set to `2014` will, with the code above, suddenly **NOT** be under the new process. You need to check that it is under 2014 or 2015.

**Comment by darobin** [comment]
Note that this has conflicts.

**Comment by marcoscaceres** [comment]
@plehegar can you please rebase?

File: js/w3c/templates/sotd.html [hunk 2]
```diff
@@ -136,7 +136,7 @@
         </p>
         {{#if isNewProcess}}
           <p>This document is governed by the <a id="w3c_process_revision"
-            href="http://www.w3.org/2014/Process-20140801/">1 August 2014 W3C Process Document</a>.
+            href="http://www.w3.org/2015/Process-20150901/">1 August 2015 W3C Process Document</a>.
```
Comment (deniak) [inline js/w3c/templates/sotd.html]: Should be `1 September 2015`

**Comment by plehegar** [comment]
rebased

**Comment by deniak** [comment]
The process 2015 is now replacing the process 2014. This MR should be merged.

**Comment by marcoscaceres** [comment]
@deniak done and released in 3.2.67.

**Comment by deniak** [comment]
:+1: @marcoscaceres

**Pull request merged**